model/distro/distro.go

```diff
@@ -346,6 +347,13 @@ func (s *PlannerSettings) GetGenerateTaskFactor() int64 {
 	return s.GenerateTaskFactor
 }
 
+func (s *PlannerSettings) GetNumDependentsFactor() int64 {
+	if s.NumDependentsFactor <= 0 {
+		return 10
```

Comment:
10 rather than 1 so we can tune both up and down from what our current behavior is.